internal/blocktx/blockchain_communication/block_message.go

```diff
@@ -0,0 +1,29 @@
+package blockchaincommunication
+
+import (
+	"io"
+
+	"github.com/libsv/go-p2p/chaincfg/chainhash"
+	"github.com/libsv/go-p2p/wire"
+)
+
+// BlockMessage only stores the transaction IDs of the block, not the full transactions
+type BlockMessage struct {
```

Comment:
This was previously part of `go-p2p`
I think we should move this to `internal/p2p`

Reply:
In favour ✋

Reply:
Yes it is in `go-p2p` but it is structure created specifically for ARC BlockTX (to use it you need to override BLOCK msg reading). That's why I created this structure here and I insist to left it here